src/main/java/SYSC4806/Category.java

```diff
@@ -0,0 +1,21 @@
+package SYSC4806;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import java.util.List;
+
+@Entity
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private long id;
+    private String name;
+    private List<LearningOutcome> learningOutcomes;
```

Comment:
I suggested as a List, but I'm not sure if this should have been an ArrayList or something.

Reply:
Or maybe a HashSet (no duplicates and order doesn't matter)?

Reply:
Wouldn't we care for order though? I am assuming for things like Categories we would order them alphabetically. But ofcourse, we could also use @OrderBy or query it that was, I guess it's a preference thing? (tried reading up on it so here: https://stackoverflow.com/questions/13101882/jpa-onetomany-list-vs-set)

Reply:
I will fix it in the pull request once we decide on something :)

Reply:
The order you are referring to is sorting which we can do in both a set and list. What I means by order in this case means ArrayList keeps the order in which elements are inserted into it, whereas HashSet does not maintain this order. @RyanWork thoughts on HashSet vs ArrayList?

Reply:
Hard to say. Generally I think we want to minimize the work that's done from our DB, since it'll have the most overhead. If we are sorting I think it should be done without the work of the DB. A hash set can be okay as long as we can sort it at the application layer. Albeit, we are not processing millions of rows so I dont think the overhead is going to get big enough that there will be any noticeable difference. For simplicity I feel like ArrayLists are better. :P agree/disagree? Doesnt matter.

Reply:
I'm cool with arraylists. I like using them more anyway :P